package proto_ff;

import "yd_fieldoptions.proto";


message E_TalentTalent
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "天赋ID"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "天赋大类"];
	optional int32 m_skillID = 3[(yd_fieldoptions.field_cname) = "技能ID"];
	optional int32 m_lvLImit = 4[(yd_fieldoptions.field_cname) = "等级上限"];
	optional int32 m_aheadSkill = 5[(yd_fieldoptions.field_cname) = "前置技能"];
	optional int32 m_aheadLv = 6[(yd_fieldoptions.field_cname) = "前置技能等级"];
	optional int32 m_needType = 7[(yd_fieldoptions.field_cname) = "要求天赋类型"];
	optional int32 m_typeNum = 8[(yd_fieldoptions.field_cname) = "要求天赋点数"];
	optional int32 m_cost = 9[(yd_fieldoptions.field_cname) = "升级消耗天赋点"];
}


message Sheet_TalentTalent
{
	repeated E_TalentTalent E_TalentTalent_List = 1[(yd_fieldoptions.field_arysize)=64];
}

message E_TalentTalenttype
{
	optional int32 m_type = 1[(yd_fieldoptions.field_cname) = "天赋大类"];
}


message Sheet_TalentTalenttype
{
	repeated E_TalentTalenttype E_TalentTalenttype_List = 1[(yd_fieldoptions.field_arysize)=4];
}
